printer/opts.go

```diff
@@ -195,3 +200,47 @@ func WithUpgradeNotice(owner, repo string, opts ...upgrade.Options) *EnableUpgra
 func (c *EnableUpgradeNotice) ApplyToContainerOption(cfg *ContainerOptions) {
 	cfg.UpgradeNotice = upgrade.NewGitHubDetector(c.owner, c.repo, c.upgradeOpts...)
 }
+
+// WithPrettyStyleFromEnv Load a custom style from environment variable
+func WithPrettyStyleFromEnv(envVariable string) (CustomPrettyStyle, error) {
+	path := os.Getenv(envVariable)
+	options, err := parseConfigFile(path)
+
+	return options, err
+}
+
+// WithPrettyStyleFile Load a custom style from file
+func WithPrettyStyleFile(path string) (CustomPrettyStyle, error) {
+	options, err := parseConfigFile(path)
+
+	return options, err
+}
+
+func parseConfigFile(fileName string) (CustomPrettyStyle, error) {
+	var options CustomPrettyStyle
+	styleConfig := style.DefaultConfig(PrettyLayoutGoTpl)
```

Comment:
When you will run the code without env value, it will panic as the returned `options` will have a nil `cfg` property. Let's return default in all cases:
```suggestion
	options := &CustomPrettyStyle{cfg: PrettyDefaultRenderConfig()}
```

Also, let's introduce a new func to explicitly return default renderer config, in file `pretty.go`, please add:
```go
// PrettyDefaultRenderConfig returns the default render configuration when no customizations are provided.
func PrettyDefaultRenderConfig() *style.Config {
	return style.DefaultConfig(PrettyLayoutGoTpl)
}
```